frontend/src/components/DeclarantIdentity.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div v-if="user">
+    <p>{{ user.firstName }} {{ user.lastName }}</p>
+    <p>
+      Adresse email :
+      <a :href="`mailto:${user.email}`" target="_blank">{{ user.email }}</a>
+    </p>
+  </div>
+  <p v-else>
+    Cette déclaration n'a pas de déclarant ou déclarante assigné. Le compte a probablement été supprimé depuis.
```

Comment:
```suggestion
    Cette déclaration n'a pas de déclarant ou déclarante assignée. Le compte a probablement été supprimé depuis.
```
petite suggestion d'après : https://doc.incubateur.net/communaute/gerer-son-produit/les-standards/design/produire-des-contenus/ecriture-inclusive#laccord-de-proximite

Reply:
Bien vu !